examples/nlp/intent_detection_slot_tagging/joint_intent_slot_infer.py

```diff
@@ -18,14 +18,30 @@
 import os
 
 import numpy as np
-from sklearn.metrics import classification_report
+from sklearn.metrics import confusion_matrix
 
 import nemo
 import nemo.collections.nlp as nemo_nlp
 from nemo import logging
 from nemo.collections.nlp.data.datasets.joint_intent_slot_dataset import JointIntentSlotDataDesc
 from nemo.collections.nlp.nm.data_layers import BertJointIntentSlotDataLayer
 from nemo.collections.nlp.nm.trainables.joint_intent_slot import JointIntentSlotClassifier
+from nemo.collections.nlp.utils.callback_utils import get_classification_report, get_f1_scores
```

Comment:
You are still calling from callback_utils.py. Can you move that part of your code which call these functions into the evaluation_utils.py?

Reply:
This 2 functions are also used in joint_intent_slot_with_bert.py callback utils. So I did not want  to make too many changes for now.